recipes/evalml-core/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: evaml-core
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/evalml/evalml-{{ version }}.tar.gz
+  sha256: 1c4c440e9370f49ad21100908285a23e7c38fbc872feba34c0e6c47c39db8f76
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy >=1.16.4
+    - pandas >=0.25.0
+    - scipy >=1.2.1,<1.5.0
+    - scikit-learn >=0.21.3,!=0.22,<0.23.0
+    - scikit-optimize
+    - tqdm >=4.33.0
+    - colorama
+    - cloudpickle >=0.2.2
+    - click >=7.0.0
+    - psutil >=5.6.3
+    - requirements-parser >=0.2.0
+    - shap >=0.35.0
+    - texttable >=1.6.2
+    - python >=3.6.*
+
+test:
+  imports:
+    - evalml
+  requires:
+    - pytest ==4.4.1
+    - nbval ==0.9.3
```

Comment:
Do these need to be so tight?

Reply:
I relaxed pytest! I'm not sure about nbval but I don't want to stray from what's listed in the package's test requirements.

Reply:
OK